lighthouse-core/lib/sd-validation/scripts/download-jsonldcontext.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * Call this script to update assets/jsonldcontext.json with the latest schema.org spec
+ */
+
+const fetch = require('isomorphic-fetch');
+const path = require('path');
+const fs = require('fs');
+
+const SCHEMA_ORG_URL = 'https://schema.org/docs/jsonldcontext.json';
```

Comment:
If you fetch with "Accept: application/ld+json" headers, you don't need to use this magic URL.  It has been documented before that this URL is for convenience only and should not be treated as an API.  Tools will generally fetch the `@context` value exactly as specified.

`curl -H "Accept: application/ld+json" https://schema.org/`

Reply:
ah neat thanks!! done 👍